src/src/WebServer/Markup_Forms.h

```diff
@@ -109,7 +121,19 @@ void addFormFloatNumberBox(const String& label,
                            const String& tooltip = EMPTY_STRING
                            #endif // ifdef ENABLE_TOOLTIPS
                            );
-void addFormNumericBox(const __FlashStringHelper * label, const __FlashStringHelper * id, int value, int min = INT_MIN, int max = INT_MAX);
+
+void addFormFloatNumberBox(const __FlashStringHelper * label,
+                           const __FlashStringHelper * id,
+                           float         value,
+                           float         min,
+                           float         max,
+                           uint8_t       nrDecimals = 6,
+                           float         stepsize   = 0.0f
```

Comment:
Wouldn't it be more useful to have a step-size of 1.0f ?

Reply:
Nope...
See:
```c++
  if (stepsize <= 0.0f) {
    html += F("0.");

    for (uint8_t i = 1; i < nrDecimals; ++i) {
      html += '0';
    }
    html += '1';
  } else {
    html += toString(stepsize, nrDecimals);
  }
```
In other words, by not setting the stepsize, you will get one based on the nr of decimals..

Not sure if that's practical though.